src/config.json

```diff
@@ -56,6 +56,12 @@
 			"ru": "Продолжить чтение"
 		},
 		"tutorial": {
+			"begin": {
+				"en": "Begin Tutorial",
+				"kr": "튜토리얼 시작",
+				"ru": "Начать обучение",
+				"zh": "开始教程"
+			},
```

Comment:
This seemed like a better fit for our config than in the individual markdown files.